### PR TITLE
update with 2021 branding | issue:#212

### PR DESCRIPTION
**Pull request by manojkrishnak**
This pull request is in regard with issue [#212](https://github.com/hacktoberfest-finder/hacktoberfest-finder/issues/212) on update hacktoberfest-finder with 2021 branding details. 

 changes made: 
- changed Favicon related files 
- changed the colors in tailwind.config.js with respect to colors in hacktoberfest official website 
- changed the logo of hacktoberfest to latest one

### CODE REVIEW

File: img/favicons/site.webmanifest [hunk 1]
```diff
@@ -1,20 +1 @@
...
-      "type": "image/png"
-    }
-  ],
-  "start_url": "https://hacktoberfest-finder.netlify.com",
-  "theme_color": "#152347",
-  "background_color": "#152347",
-  "display": "standalone"
-}
\ No newline at end of file
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (duncanmcclean) [inline img/favicons/site.webmanifest:1]: Please can you format this file in the same way it was before? It's easier for me to review what's changed if it's kept the same.

Thanks!

File: tailwind.config.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ module.exports = {
   theme: {
     extend: {
         colors: {
-            'hacktoberfest-blue': '#0069ff',
-            'hacktoberfest-orange': '#9c4668'
+            'hacktoberfest-blue': '#91A88C',
```
Comment (duncanmcclean) [inline tailwind.config.js:10]: This colour is green, please replace `hacktoberfest-blue` with `hacktoberfest-green`. 

image

File: tailwind.config.js [hunk 3]
```diff
@@ -7,8 +7,8 @@ module.exports = {
...
+            'hacktoberfest-orange': '#B53A25'
```
Comment (duncanmcclean) [inline tailwind.config.js:11]: Same here, can you update `hacktoberfest-orange` to something like `hacktoberfest-red`?

image

**Review by duncanmcclean — CHANGES_REQUESTED** [review]
Thanks! Just made a couple comments..

**Comment by manojkrishnak** [comment]
Made changes as per the comments mentioned. Please check.

File: img/favicons/site.webmanifest [hunk 4]
```diff
@@ -1,20 +1,19 @@
 {
-  "name": "Hacktoberfest Finder",
-  "short_name": "Hacktoberfest",
+  "name": "",
```
Comment (duncanmcclean) [inline img/favicons/site.webmanifest:2]: Would you mind filling this back in to be `Hacktoberfest Finder`, same with `short_name`.

**Review by duncanmcclean — CHANGES_REQUESTED** [review]
Thanks! Sorry to be nit-picky but I have a few other changes in the manifest.

**Comment by manojkrishnak** [comment]
can you check now?

**Review by duncanmcclean — APPROVED** [review]
Looks good, thanks!

**Pull request merged**